0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -78,24 +78,35 @@ If some transfers in an Interledger payment are executed and others expire, the
 
 Failing to deliver the fulfillment in time is the main risk connectors face and there are a number of additional strategies connectors should employ to mitigate and manage this risk. For more details, see [IL-RFC 18](../0018-connector-risk-mitigations/0018-connector-risk-mitigations.md).
 
-
 ## Interledger Protocol Suite
 
-The Interledger stack is separated into four layers:
+The Interledger stack is separated into the following layers:
 
 ![Interledger Protocol Suite](../shared/graphs/protocol-suite.svg)
 
-### Ledger Layer
+**Note:** What was formerly known as the "Ledger Layer" is now split into the Settlement and Clearing Layers.
+
+### Settlement Layer
+
+In order for two parties to transact, they must be able to send and receive value in the same settlement system. Examples of settlement systems include bank ledgers, closed-loop payment networks, cryptocurrencies, blockchains, or cash.
+
+Settlement systems may or may not have APIs, and the features that they support determine the type of clearing systems that can be used with them.
+
+### Clearing Layer
+
+It is often too slow or expensive to settle every payment via the underlying settlement system. In these cases, a clearing protocol may be used on top of the underlying settlement mechanism. Clearing protocols may be implemented on bilateral or multilateral bases, such as a trustline or a payment channel network like the [Lightning Network](https://lightning.network), respectively.
```

Comment:
Maybe mention that the clearing layer can also be more complex, like the one used in ilp-plugin-xrp-paychan, where the receiver trusts the sender to send an *unconditional* claim update after fulfillment of the otherwise conditional transfer.